packages/skipper/tests/FileSystem/PathNormalizerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
```

Comment:
Better create some file instead of `.gitkeep`. It better reflects real life.

I can't even comment on it now :D